extensions/ql-vscode/src/common/interface-types.ts

```diff
@@ -596,6 +596,7 @@ export type ToModelEditorMessage =
   | RevealMethodMessage;
 
 export type FromModelEditorMessage =
+  | CommonFromViewMessages
   | ViewLoadedMsg
```

Comment:
I believe this is included in the `CommonFromViewMessages`, so we should be able to remove this.